core/templates/dev/head/pages/admin/admin.html

```diff
@@ -19,406 +26,38 @@
-      {% if DEV_MODE %}
-        <div class="col-lg-12 col-md-12 col-sm-12"
-             ng-if="currentTab === ADMIN_TAB_URLS.ACTIVITIES"
-             style="margin-top: 16px;">
-          <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
-            <div class="container" style="margin-left: 0;">
-              <h3>Reload a single exploration</h3>
-              {% for exploration in demo_explorations %}
-                <div class="row protractor-test-reload-exploration-row">
-                  <span class="col-lg-4 col-md-4 col-sm-4 protractor-test-reload-exploration-title">
-                    {{exploration[1]}}
-                  </span>
-                  <span class="col-lg-2 col-md-2 col-sm-2">
-                    <button type="button"
-                            class="btn btn-default protractor-test-reload-exploration-button pull-right"
-                            ng-click="reloadExploration({{exploration[0]}})">
-                    Reload
-                    </button>
-                  </span>
-                </div>
-              {% endfor %}
-              <button type="button"
-                      class="btn btn-default protractor-test-reload-all-explorations-button"
-                      ng-click="reloadAllExplorations()">
-                Reload All Explorations
-              </button>
-            </div>
-          </md-card>
-
-          <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
-            <div class="container" style="margin-left: 0;">
-              <h3>Reload a single collection</h3>
-              {% for collection in demo_collections %}
-                <div class="row protractor-test-reload-collection-row">
-                  <span class="col-lg-4 col-md-4 col-sm-4 protractor-test-reload-collection-title">
-                    {{collection[1]}}
-                  </span>
-                  <span class="col-lg-2 col-md-2 col-sm-2">
-                    <button type="button"
-                            class="btn btn-default protractor-test-reload-collection-button pull-right"
-                            ng-click="reloadCollection({{collection[0]}})">
-                      Reload
-                    </button>
-                  </span>
-                </div>
-              {% endfor %}
-            </div>
-          </md-card>
-        </div><br><br>
-      {% else %}
-        <div class="col-lg-12 col-md-12 col-sm-12"
-             ng-if="currentTab === ADMIN_TAB_URLS.ACTIVITIES"
-             style="margin-top: 16px;">
-          <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
-            The 'Activities' tab is not available in the production environment.
-          </md-card>
-        </div>
-      {% endif %}
+    <div style="padding-top: 16px;">
+      <admin-dev-mode-activities-tab ng-if="isActivitiesTabOpen() && inDevMode"
+                            set-status-message="setStatusMessage">
```

Comment:
I think this needs to be indented a bit more; you probably wrote it when the tag name was still admin-activities-tab, I'm guessing :)